recipes/nss/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "3.36.1" %}
+{% set version_us = version.replace(".", "_") %}
+
+package:
+  name: nss
+  version: {{ version }}
+
+source:
+  patches: nss-3.36.1-standalone-1.patch
```

Comment:
Could you please move this below the source and format like this? Also filling out a comment explaining the patch would help.

```yaml
  patches:
    # <add description here>
    - nss-3.36.1-standalone-1.patch
```

Reply:
Got it, going to change